keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -63,7 +63,7 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 ## Summary
 
-Add support for a drop-in configuration directory for the Kubelet. This directory will be in `/etc/kubernetes/kubelet.conf.d` and configuration files will be processed in alphanumeric order. Establishment of conventions for configuration processing will be done, and further work can be done to add this support for other components.
+Add support for a drop-in configuration directory for the Kubelet. This directory can be specified via a "--config-dir" flag, and configuration files will be processed in alphanumeric order. Establishment of conventions for configuration processing will be done, and further work can be done to add this support for other components.
```

Comment:
nit: somewhere we should establish the default for the flag will be `/etc/kubernetes/kubelet.conf.d`

Reply:
Or do we want to follow the same convention as for `--config` (empty by default) so that if `--config-dir` is not specified drop-in support is disabled?

Reply:
ah good point, then we should mention that

Reply:
> Or do we want to follow the same convention as for `--config` (empty by default) so that if `--config-dir` is not specified drop-in support is disabled?

+1 to align with --config defaults

Reply:
updated, thanks!